deploy/deploy.sh

```diff
@@ -85,14 +85,28 @@ delete() {
 
 # create the objects
 if [ "$method" == "CREATE" ]; then
-    echo "  * [ii] Creating the operators" ; create ${operator_list[@]} && sleep 30
+    echo "  * [ii] Creating the operators" ; create ${operator_list[@]}
+    echo ""
+    echo "+-----------------------------------------------------------------------+"
+    echo "| Press Ctrl+C when prometheus-operator-1-<unique> is marked as Running |"
+    echo "+-----------------------------------------------------------------------+"
+    echo ""
+    trap ' ' INT
+    oc get pods -w
```

Comment:
feel this could be a background task or perhaps we can add a verbose option and show only if thats passed. Not a blocker we can revisit later.